### PR TITLE
Add Clover

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

**Comment by phallobst** [comment]
This one might be better fitted to _Other_, I guess.

**Comment by 40504** [comment]
I saw all content about payments, thought it's suitable for payments category

**Comment by 40504** [comment]
moved to other category

File: _data/other.yml [hunk 1]
```diff
@@ -116,6 +116,15 @@ websites:
     tfa:
       - totp
 
+  - name: Clover
+    url: https://www.clover.com
+    img: clover.png
+    tfa:
+      - totp
```
Comment (ApCoder123) [inline _data/other.yml:123]:
```suggestion
      - totp
      - sms
```
The doc page mentions SMS 2FA as well.

**Review by ApCoder123 — CHANGES_REQUESTED** [review]
Please use this SVG as the icon:
clover svg

File: _data/other.yml [hunk 2]
```diff
@@ -116,6 +116,16 @@ websites:
...
 
+  - name: Clover
+    url: https://www.clover.com
+    img: clover.svg
+    tfa:
+      - totp
+      - sms
+    doc: https://www.clover.com/help/two-factor-authentication
+    regions:
+      - us
```
Comment (ApCoder123) [inline _data/other.yml:127]:
```suggestion
      - us
      - ca
      - gb
      - ie
      - de
      - at
      - nl
      - ar
```
The site is available in more regions than just the US - e.g. https://www.la.clover.com/ar/ayuda/autenticaci%C3%B3n-de-dos-factores/

**Review by ApCoder123 — CHANGES_REQUESTED** [review]

File: _data/other.yml (hunk 2 above)
Comment (40504) [reply _data/other.yml:127]: Done ✅

**Review by ApCoder123 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**